IPython/html/services/kernels/handlers.py

```diff
@@ -289,7 +275,5 @@ class StdinHandler(ZMQChannelHandler):
     (r"/api/kernels", MainKernelHandler),
     (r"/api/kernels/%s" % _kernel_id_regex, KernelHandler),
     (r"/api/kernels/%s/%s" % (_kernel_id_regex, _kernel_action_regex), KernelActionHandler),
-    (r"/api/kernels/%s/iopub" % _kernel_id_regex, IOPubHandler),
-    (r"/api/kernels/%s/shell" % _kernel_id_regex, ShellHandler),
-    (r"/api/kernels/%s/stdin" % _kernel_id_regex, StdinHandler)
+    (r"/api/kernels/%s/channels" % _kernel_id_regex, ZMQChannelsHandler),
```

Comment:
@rgbkrk depends on your terminology. We generally refer to shell, iopub, stdin as 'channels' meaning the different categories/namespaces of messages, so there are still three channels. The change here is that those three channels all share the same websocket, though, so I think the answer to your question is "Technically no, but yes."